templates/interfaces.j2

```diff
@@ -183,7 +202,8 @@ post-up ip route replace {{ route_net }}
 	{%- if
 	route_options.mtu is defined or
 	network_management_default_mtu is defined
-	%} mtu {{ route_options.mtu|default(network_management_default_mtu) }}{%- endif %}{{ "\n" }}
+	%} mtu {{ route_options.mtu|default(network_management_default_mtu) }}{%- endif %}
+	{%- if route_options.table is defined %} table {{ route_options.table }}{%- endif %}{{ "\n" }}
```

Comment:
`route_options.table` is not documented in README.md

Reply:
Now it is.